src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
```

Comment:
It might make sense to use the [`organization_name`](https://github.com/googleapis/googleapis/blob/master/gapic/api/artman_datastore.yaml#L6) configuration, then. (Just replace the hyphen with the file separator.) You probably will need to modify the [package metadata generation task](https://github.com/googleapis/artman/blob/master/artman/tasks/package_metadata_tasks.py#L26) in artman to pass this through to toolkit.